lib/bootstrap_builders/flash.rb

```diff
@@ -0,0 +1,44 @@
+class BootstrapBuilders::Flash
+  def initialize(args)
+    @class = args[:class]
+    @alert_types = [:success, :info, :warning, :danger]
+    @context = args[:context]
```

Comment:
If an argument is required, you should use `.fetch` instead of `.[]`. That will raise a proper error, telling the user that the argument is missing.

The alternative is a nil-error later, which can be hard to debug and figure out.